src/boot/efi/boot.c

```diff
@@ -2174,6 +2218,108 @@ static void config_entry_add_linux(
                                 return (void) log_oom();
                 }
         }
+
+        uefi_call_wrapper(linux_dir->Close, 1, linux_dir);
+}
+
+static EFI_STATUS secure_boot_enroll(EFI_FILE *root_dir, Config *config, ConfigEntry *entry) {
+        assert(root_dir);
+        assert(config);
+        assert(entry);
+
+        typedef struct {
+                const CHAR16 *name;
+                const CHAR16 *filename;
+                const EFI_GUID vendor;
+                CHAR8 *buffer;
+                UINTN size;
+        } SecureBootVariable;
+
+        static SecureBootVariable sb_vars[] = {
+                { L"db",  L"db.esl",  EFI_IMAGE_SECURITY_DATABASE_VARIABLE, NULL, 0 },
+                { L"KEK", L"KEK.esl", EFI_GLOBAL_VARIABLE, NULL, 0 },
+                { L"PK",  L"PK.esl",  EFI_GLOBAL_VARIABLE, NULL, 0 },
+        };
+
+        EFI_STATUS err;
+        _cleanup_freepool_ CHAR16 *keys_dir = NULL;
+        _cleanup_(FileHandleClosep) EFI_FILE_HANDLE dir = NULL;
+        UINT32 sb_vars_opts =
+                EFI_VARIABLE_NON_VOLATILE |
+                EFI_VARIABLE_BOOTSERVICE_ACCESS |
+                EFI_VARIABLE_RUNTIME_ACCESS |
+                EFI_VARIABLE_TIME_BASED_AUTHENTICATED_WRITE_ACCESS;
+
+        keys_dir = PoolPrint(L"\\loader\\keys\\%s", entry->current_name);
+        if (!keys_dir) return log_oom();
```

Comment:
we generally line break after the condition of the `if`, i.e. insert a line break before the `return` please.